blivet/devices/lvm.py

```diff
@@ -1523,14 +1523,14 @@ def _update_format_from_origin(self):
             fmt._chrooted_mountpoint = None
             fmt.device = self.path  # pylint: disable=no-member
 
-        super(LVMSnapShotBase, self)._set_format(fmt)
+        super(LVMSnapShotBase, self)._set_format(fmt)  # pylint: disable=no-member
```

Comment:
IMHO to be Python3 compatible it would be nice to change this ``super(LVMSnapShotBase, self)`` to just ``super()``.